resources/js/components/modules/user/index.vue

```diff
@@ -1,112 +1,180 @@
+                                                :class="[
+                                                    page == pageNumber
+                                                        ? 'active'
+                                                        : '',
+                                                ]"
+                                            >
+                                                <a
+                                                    href=""
+                                                    class="page-link"
+                                                    @click.prevent="
+                                                        selectPage(page)
+                                                    "
+                                                >
+                                                    {{ page + 1 }}</a
+                                                >
+                                            </li>
+                                            <li
+                                                class="page-item"
+                                                v-if="
+                                                    pageNumber < pageCount - 1
+                                                "
+                                            >
+                                                <a
+                                                    href=""
+                                                    class="page-link"
+                                                    @click.prevent="nextPage"
+                                                    >Post</a
+                                                >
+                                            </li>
+                                        </ul>
+                                    </div>
+                                </template>
+                                <template v-else>
+                                    <div class="callout callout-info">
+                                        <h5>No se encontraron resultados...</h5>
+                                    </div>
+                                </template>
                             </div>
-                        </template>
                         </div>
-                     </div>
+                    </div>
                 </div>
             </div>
         </div>
     </div>
-</div>
 </template>
 
 <script>
-import Form from './Form.vue'
+import Form from "./Form.vue";
 export default {
-    data(){
+    data() {
         return {
             listUsers: [],
             pageNumber: 0,
             perPage: 5,
-        }
+        };
     },
-    mounted(){
-        this.getlistUsers(this.$store.state.user.role_id, this.$store.state.user.client_id);
+    mounted() {
+        this.getlistUsers(
+            this.$store.state.user.role_id,
+            this.$store.state.user.client_id
+        );
     },
 
-    props:['user'],
+    props: ["user"],
```

Comment:
Codacy has a fix for the issue: [Unexpected blank line between object properties.](https://app.codacy.com/gh/JarvinsonValencia/easyConference/pullRequest?prid=9473680)
```suggestion

```